src/main/resources/conf/application.yml

```
@@ -0,0 +1,6 @@
+# 阿里云短信认证信息
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)